ebuild-writing/variables/text.xml

```diff
@@ -568,23 +632,77 @@ not be listed.
 </p>
 
 </body>
-</section>
+</subsection>
 
-<section>
-<title>LICENSE</title>
+<subsection>
+<title>REQUIRED_USE</title>
 <body>
+<p>
+The <c>REQUIRED_USE</c> variable contains a list of assertions that must be
+met by the configuration of USE flags to be valid for this ebuild. In order
+to be matched, a USE flag in a terminal element must be enabled
+(or disabled if it has an exclamation mark prefix).
+</p>
 
 <p>
-It is possible to specify multiple <c>LICENSE</c> entries, and entries which only
-apply if a particular <c>USE</c> flag is set. The format is the same as for
-<c>DEPEND</c>. See <uri link="https://www.gentoo.org/glep/glep-0023.html">
-GLEP 23</uri> for details.
+Essentially, <c>REQUIRED_USE</c> is an analogue of <c>DEPEND</c> style syntax.
+For example, to state that some combination is forbidden, i.e. "if foo is set,
+bar must be unset":
 </p>
 
+<codesample lang="ebuild">
+REQUIRED_USE="foo? ( !bar )"
+</codesample>
+<p>
+To state "if foo is set, then at least one of bar, baz, and quux must be activated":
+</p>
+<codesample lang="ebuild">
+REQUIRED_USE="foo? ( || ( bar baz quux ) )"
+</codesample>
+<p>
+To state "exactly one of foo, bar, or baz must be set, but not several":
+</p>
+<codesample lang="ebuild">
+REQUIRED_USE="^^ ( foo bar baz )"
+</codesample>
+<p>
+Note that the last relationship is that of an Exclusive OR (XOR).
+While an XOR could be formed from usual DEPEND syntax,
```

Comment:
I don't really like "formed from", what do you think of this?
```suggestion
While an XOR could be formed with the usual DEPEND syntax,
```

Reply:
Note that this is simply a reordering of sections while the wording was there before.

IMHO we can fix obvious small problems here (like the whitespace in the `ROOT` example), but other unrelated issues should go to their own PR.

BTW, "formed from" is my own wording from [bug 353624 comment 8](https://bugs.gentoo.org/353624#c8). :smile: